Firestore/core/src/bundle/bundle_serializer.cc

```diff
@@ -210,7 +211,7 @@ Filter InvalidFilter() {
   // The exact value doesn't matter. Note that there's no way to create the base
   // class `Filter`, so it has to be one of the derived classes.
   return FieldFilter::Create({}, {},
-                             SharedMessage<google_firestore_v1_Value>({}));
+                             MakeSharedMessage<google_firestore_v1_Value>({}));
```

Comment:
Nit: perhaps it would be slightly more readable as:
```cpp
MakeSharedMessage(google_firestore_v1_Value{}));
```

Reply:
Agreed and fixed.